en/components/bulletgraph.md

```diff
@@ -21,9 +23,8 @@ This gauge supports only one scale, one set of tick marks and one set of labels.
 
 <div class="divider--half"></div>
 
-The bullet graph component provides you with the ability to create attractive data presentations, replacing meters and gauges that are used on dashboards with simple yet straightforward and clear bar charts. A bullet graph is one of the most effective and efficient ways to present progress towards goals, good/better/best ranges, or compare multiple measurements in as little horizontal or vertical space as possible.
-
-The features of the bullet graph include configurable orientation and direction, configurable visual elements, and more. The control has also a built-in support for animated transitions.
+This gauge supports one scale, one set of tick marks and one set of labels. The component also has built-in support for animated transitions. This animation is easily customizable by setting the `transitionDuration` property. 
+The features of the bullet graph include configurable orientation and direction, configurable visual elements such as the needle, and more. 
 
```

Comment:
perhaps, we should have a new section/sample that shows orientation and direction properties?